### ISSUE TITLE
Етап 1

### ISSUE THREAD

**Issue by AsAndrew** [AsAndrew/Machine-Learning#1]
Знайти готовий продукт.
Навчання без вчителя.
Гра Space Drifters 2D
Підібрати бібліотеку для навчання нейронної мережі.

**Comment by QIzxqzy** [comment]
http://itc.ua/news/ii-udalos-obyigrat-v-poker-spetsialistov-v-oblasti-iskusstvennogo-intellekta/
а ведь и вправду, потом только в онлайн казино закинуть)

**Comment by AsAndrew** [comment]
Це приклад нейронної мережі і numpy.
https://habrahabr.ru/post/271563/

**Comment by QIzxqzy** [comment]
не совсем понимаю как происходит взаимодействие монитора с нейронной сетью

**Comment by QIzxqzy** [comment]
https://github.com/g0t0wasd/python/blob/master/games/arcade_games/pong.py
можно для начала взять игру пинг-понг, а потом что то серьезнее

**Comment by QIzxqzy** [comment]
да, и выбранная нами игра слишком монотонная. Предлогаю сменить ее на http://store.steampowered.com/app/497670/?snr=1_7_7_151_150_1
Line / Dash

в крайнем случае - http://store.steampowered.com/app/498330/?snr=1_7_7_151_150_1

**Comment by AsAndrew** [comment]
https://habrahabr.ru/post/271563/
Спробуй запустити ці мержі.



2017-04-17 12:26 GMT+03:00 QIzxqzy <notifications@github.com>:
